libcxx/test/std/utilities/expected/expected.void/observers/has_value.pass.cpp

```diff
@@ -43,6 +44,17 @@ constexpr bool test() {
     assert(!e.has_value());
   }
 
+  // See comments of the corresponding test in
+  // "expected.expected/observers/has_value.pass.cpp".
+  {
+    const std::expected<void, TailClobberer<1>> e(std::unexpect);
+    // clang-cl does not support [[no_unique_address]] yet.
+#if !(defined(TEST_COMPILER_CLANG) && defined(_MSC_VER))
+    static_assert(sizeof(TailClobberer<1>) == sizeof(e));
+#endif
```

Comment:
`[[no_unique_address]]` is a libc++ extension to `std::expected`.  
we should put this test in test/libcxx folder
(tests in test/std is meant to pass for all stl implementations)

Reply:
Same here, I recommend `LIBCPP_STATIC_ASSERT`.

Reply:
Done!